packages/phoenix-event-display/src/managers/three-manager/import-manager.ts

```diff
@@ -256,9 +256,9 @@ export class ImportManager {
                 });
               });
             } catch (error) {
-              this.eventDisplay
-                .getInfoLogger()
-                .add('Could not read zip file', 'Error');
+              // this.eventDisplay
+              //   .getInfoLogger()
+              //   .add('Could not read zip file', 'Error');
```

Comment:
Should this be uncommented ? If not, why not drop it ?

Reply:
Oops. I missed this. It caused an error, so I think we just need to delete it (I was hoping to find a solution and so left it in as a reminder).